interfaces/IERC777Sender.sol

```diff
@@ -0,0 +1,36 @@
+//Copyright (c) 2019 Clearmatics Technologies Ltd
+
+// SPDX-License-Identifier: LGPL-3.0+
+
+pragma solidity ^0.5.0;
+
+/**
+* @title ERC777 Token Sender Interface
+* @author Andrea Di Nenno
+* @notice This interface allows a token sender (EOA or Smart Contract) to add extra logic 
+* to control his outgoing tokens.
+*/
+
+interface IERC777Sender {
+
+    /**
+    * @notice Called by a `ERC777` token contract before tokens are being transferred or burned.
```

Comment:
Called by an `ERC777` token contract before tokens are transferred or burned.

Don't think the being is needed here.